WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java

```diff
@@ -216,11 +236,34 @@ public View getView(int position, View convertView, ViewGroup parent) {
             holder.entryTextView.setTextColor(context.getResources().getColor(R.color.stats_text_color));
             holder.rowContent.setClickable(false);
 
+            final FollowDataModel followData = currentRowData.getFollowData();
+
             // entries
-            if (mTopPagerSelectedButtonIndex == 0) {
-                // WPCOM followers
-                holder.entryTextView.setText(currentRowData.getLabel());
-                if (!TextUtils.isEmpty(currentRowData.getURL())) {
+            if (mTopPagerSelectedButtonIndex == 0 && !TextUtils.isEmpty(currentRowData.getURL())) {
+                // WPCOM followers with no empty URL
+
+                boolean openInReader = true;
+                if (followData == null) {
+                    // If follow data is empty, we cannot follow the blog, or access it in the reader.
+                    // We need to check if the user is a member of this blog.
+                    // If so, we can launch open the reader, otherwise open the blog in the in-app browser.
+                    openInReader = false;
```

Comment:
We should compare the URL of the blog, since Jetpack blogs can live in subfolders.